source/android/README.md

```diff
@@ -25,6 +25,12 @@ swig -c++ -java -package io.adaptivecards.objectmodel -outdir main/java/io/adapt
 2. Select folder "<github directory>\AdaptiveCards\source\android"
 3. Build->"Rebuild Project"
 
+##or on a Mac
+ ```console
+ cd source/android
+ make
+ ```
+
 ## Build failure "UnsupportedClassVersionError 52.0"
 
 If you get a build failure similar to "UnsupportedClassVersionError 52.0" then from Android Studio go to: **File** -> **Other Settings** --> **Default project structure...** --> Check **Use Embedded JDK**
```

Comment:
were you able to get to the bottom of the long-filename related build failures? Should those be in the readme?

Reply:
By renaming the folder where the project is located I didn't had to face the issues again, the only caveat I found while making the sample app is that having our package contain Android in its name you end up having to write global::Android at some points but nothing else

Reply:
Might be a good idea to document the error/symptoms you were observing in some sort of "Troubleshooting" section 

---
In reply to: [348199717](https://github.com/microsoft/AdaptiveCards/pull/3610#discussion_r348199717) [](ancestors = 348199717)

Reply:
Should this be on the OneNote or in a different text file?

---
In reply to: [349356400](https://github.com/microsoft/AdaptiveCards/pull/3610#discussion_r349356400) [](ancestors = 349356400,348199717)